src/pages/HomeSigningOfficial.js

```diff
@@ -80,7 +80,7 @@ class HomeSigningOfficial extends Component {
               div({style: {marginTop: '2rem', marginBottom: '2rem'}},
                 [a({
                   id: 'link_downloadAgreement',
-                  href: '/Data_Provider_Agreement.pdf',
+                  href: '/DUOS_LCA_01.2021.pdf',
```

Comment:
The old file is referenced in 4 places. Instead of adding a new one in two of the 4, can we instead just replace the contents of `Data_Provider_Agreement.pdf` with the new one?